src/main/java/model/Cars.java

```diff
@@ -0,0 +1,63 @@
+package model;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class Cars {
+    private final List<Car> cars;
+
+    public Cars (List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public List<Car> stepForward () {
+        cars.forEach(car -> car.stepForward(new RandomMovingStrategy()));
+        return cars;
+    }
+
+    public void printCars () {
+        cars.forEach(Car::printCar);
+    }
+
+    public Position getMaxPosition () {
+        Position maxPosition = new Position(0);
+
+        for (Car car : cars) {
+            maxPosition = car.getMaxPosition(maxPosition);
+        }
+
+        return maxPosition;
+    }
+
+    public List<Car> getWinners () {
+        List<Car> winners;
+
+        Position maxPosition = getMaxPosition();
+        winners = cars.stream().filter(car -> car.isWinner(maxPosition)).collect(Collectors.toList());
```

Comment:
```suggestion
        winners = cars.stream()
        .filter(car -> car.isWinner(maxPosition))
        .collect(Collectors.toList());
```
스트림은 기능단위로 줄바꿈하여 사용하는것을 권장 드려요.